.gitmodules

```diff
@@ -1,12 +1,14 @@
 [submodule "modules/native-android"]
 	path = modules/native-android
-	url = https://github.com/hashcube/native-android.git
+	url = https://github.com/CodeCombiner/native-android.git
 [submodule "modules/native-ios"]
 	path = modules/native-ios
 	url = https://github.com/hashcube/native-ios.git
+	branch = hc
```

Comment:
why do we need this?

Reply:
will this take latest hc automatically? if yes better to remove this. I don't think we always use latest hc.

Reply:
Removed hc branch pointer from all git submodules